README.md

```diff
@@ -1,6 +1,6 @@
 ![freeCodeCamp Guide](https://s3.amazonaws.com/freecodecamp/wide-social-banner.png)
 
-# The freeCodeCamp Guide
+<h1 align="center"> The freeCodeCamp Guide </h1>
```

Comment:
Hi @sayantanHack 

Thank you for opening this PR. Please note that HTML formatting should not be used in markdown files. I will be closing this to keep the current markdown formatting.

Please feel free to open a more substantial PR.

Reply:
Thank you